doc/api/vm.md

```diff
@@ -736,8 +736,8 @@ added:
 
 > Stability: 1 - Experimental
 
-*This feature is only available with the `--experimental-vm-modules` command
-flag enabled.*
+*This feature is only available with the `--experimental-vm-modules`
+command-flag enabled.*
```

Comment:
These updates should include `-line`, right?

Reply:
Whoops! Thanks. Fixed.